Prog/reshape ACS data for population projection.do

```diff
@@ -0,0 +1,568 @@
+gen ftot35to39 = HC06_EST_VC10*HC01_EST_VC01/100
+gen tot40to44 = HC02_EST_VC11*HC01_EST_VC01/100
+gen mtot40to44 = HC04_EST_VC11*HC01_EST_VC01/100
+gen ftot40to44 = HC06_EST_VC11*HC01_EST_VC01/100
+gen tot45to49 = HC02_EST_VC12*HC01_EST_VC01/100
+gen mtot45to49 = HC04_EST_VC12*HC01_EST_VC01/100
+gen ftot45to49 = HC06_EST_VC12*HC01_EST_VC01/100
+gen tot50to54 = HC02_EST_VC13*HC01_EST_VC01/100
+gen mtot50to54 = HC04_EST_VC13*HC01_EST_VC01/100
+gen ftot50to54 = HC06_EST_VC13*HC01_EST_VC01/100
+gen tot55to59 = HC02_EST_VC14*HC01_EST_VC01/100
+gen mtot55to59 = HC04_EST_VC14*HC01_EST_VC01/100
+gen ftot55to59 = HC06_EST_VC14*HC01_EST_VC01/100
+gen tot60to64 = HC02_EST_VC15*HC01_EST_VC01/100
+gen mtot60to64 = HC04_EST_VC15*HC01_EST_VC01/100
+gen ftot60to64 = HC06_EST_VC15*HC01_EST_VC01/100
+gen tot65to69 = HC02_EST_VC16*HC01_EST_VC01/100
+gen mtot65to69 = HC04_EST_VC16*HC01_EST_VC01/100
+gen ftot65to69 = HC06_EST_VC16*HC01_EST_VC01/100
+gen tot70to74 = HC02_EST_VC17*HC01_EST_VC01/100
+gen mtot70to74 = HC04_EST_VC17*HC01_EST_VC01/100
+gen ftot70to74 = HC06_EST_VC17*HC01_EST_VC01/100
+gen tot75to79 = HC02_EST_VC18*HC01_EST_VC01/100
+gen mtot75to79 = HC04_EST_VC18*HC01_EST_VC01/100
+gen ftot75to79 = HC06_EST_VC18*HC01_EST_VC01/100
+gen tot80to84 = HC02_EST_VC19*HC01_EST_VC01/100
+gen mtot80to84 = HC04_EST_VC19*HC01_EST_VC01/100
+gen ftot80to84 = HC06_EST_VC19*HC01_EST_VC01/100
+gen tot85up = HC02_EST_VC20*HC01_EST_VC01/100
+gen mtot85up = HC04_EST_VC20*HC01_EST_VC01/100
+gen ftot85up = HC06_EST_VC20*HC01_EST_VC01/100
+
+collapse (sum) HC01_EST_VC01 HC03_EST_VC01 HC05_EST_VC01 totunder5 mtotunder5 ftotunder5 tot5to9 mtot5to9 ftot5to9 tot10to14 mtot10to14 ftot10to14 ///
+              tot15to19  mtot15to19 ftot15to19 tot20to24 mtot20to24 ftot20to24 tot25to29 mtot25to29 ftot25to29 ///
+			  tot30to34  mtot30to34 ftot30to34 tot35to39 mtot35to39 ftot35to39 tot40to44 mtot40to44 ftot40to44 ///
+			   tot45to49  mtot45to49 ftot45to49 tot50to54 mtot50to54 ftot50to54 tot55to59 mtot55to59 ftot55to59 ///
+			   tot60to64 mtot60to64 ftot60to64 tot65to69 mtot65to69 ftot65to69 tot70to74 mtot70to74 ftot70to74 ///
+			   tot75to79 mtot75to79 ftot75to79 tot80to84 mtot80to84 ftot80to84 tot85up mtot85up ftot85up, by (Year)
+			   
+rename HC03_EST_VC01 HC02_EST_VC01
+rename HC05_EST_VC01 HC03_EST_VC01
+			   
+append using "D:\DCData\Libraries\RegHsg\Prog\08-16 ACS 1year.dta", force
+
+sort Year
+
+rename HC01_EST_VC01 totpop
+rename HC02_EST_VC01 totmalepop
+rename HC03_EST_VC01 totfemalepop
+
+xpose, clear varname format
+
+order _varname v1-v11
+
+save "D:\DCData\Libraries\RegHsg\Prog\ACS_1year_gender.dta"
+
+export excel using "D:\DCData\Libraries\RegHsg\Prog\ACS_1year_gender.xlsx"
+
+clear
+
+***************************population by gender 5 year estimate ****************
+
+import excel "L:\Libraries\RegHsg\Raw\ACS 5 year estimate\All_5year.xlsx", sheet("Sheet1") firstrow
+
+label variable HC01_EST_VC01 "totpop"
+label variable HC02_EST_VC01 "mtotpop"
```

Comment:
i don't think these are right. hc02_est_vc01 is the percent of the total population according to the metadata.  file:///L:\Libraries\RegHsg\Raw\ACS%201%20year%20estimate\aff_download\ACS_17_1YR_S0101_metadata.csv